optimum/exporters/onnx/base.py

```diff
@@ -206,6 +207,17 @@ def is_torch_support_available(self) -> bool:
             return TORCH_VERSION >= self.MIN_TORCH_VERSION
         return False
 
+    @property
+    def torch_to_onnx_input_map(self) -> Mapping[str, str]:
```

Comment:
I would make it clear that it is needed when the dummy input names and the exported input names do not match.

Reply:
Updated